lib/game_teams_manager.rb

```diff
@@ -89,7 +88,6 @@ def find_all_home_teams
   end
 
   def home_games_by_team_id
```

Comment:
Yeah we'll have to change this a little once we're ready, to get it faster than 6min. Nice work!